entities/PlayerActions.js

```diff
@@ -28,10 +27,11 @@ module.exports = class PlayerActions {
     );
   }
 
-  static prompt({ UI }, gameState) {
-    const actions = this.refresh(gameState);
+  static prompt({ UI }, gameState, actions = []) {
+    if (actions.length === 0) actions = this.refresh(gameState);
```

Comment:
Should this be the behaviour ? Override actions ? I would thing (at least for `PlayerActions#prompt`) that additional actions are concatenated ...

I know we talked about revisiting the prompts so I am fine with this as a placeholder also.

One more thing, though -> if we're leaving it this way we can actually do this instead:
```javascript
static prompt ({ UI }, gameState, actions = this.refresh(gameState)) { /**/ }
```